ncp.c

```diff
@@ -3522,6 +3517,7 @@ socket_to_conn_handler(void *arg)
     }
     break;
   }
+  return NULL;
```

Comment:
"Shouldn't get here", but this fixes the warning about reaching the end of the function without returning something.